samples/snippets/src/test/java/com/example/spanner/DmlReturningSampleIT.java

```diff
@@ -0,0 +1,137 @@
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.KeySet;
+import com.google.cloud.spanner.Mutation;
+import com.google.common.collect.ImmutableList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.concurrent.TimeUnit;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration tests for DML Returning samples for GoogleStandardSql dialect. */
+@RunWith(JUnit4.class)
+public class DmlReturningSampleIT extends SampleTestBase {
+
+  private static DatabaseId databaseId;
+
+  @BeforeClass
+  public static void createTestDatabase() throws Exception {
+    final String database = idGenerator.generateDatabaseId();
+    databaseAdminClient
+        .createDatabase(
+            instanceId,
+            database,
+            ImmutableList.of(
+                "CREATE TABLE Singers ("
+                    + "  SingerId   INT64 NOT NULL,"
+                    + "  FirstName  STRING(1024),"
+                    + "  LastName   STRING(1024),"
+                    + "  SingerInfo BYTES(MAX),"
+                    + "  FullName STRING(2048) AS "
+                    + "  (ARRAY_TO_STRING([FirstName, LastName], \" \")) STORED"
+                    + ") PRIMARY KEY (SingerId)",
+                "CREATE TABLE Albums ("
+                    + "  SingerId     INT64 NOT NULL,"
+                    + "  AlbumId      INT64 NOT NULL,"
+                    + "  AlbumTitle   STRING(MAX),"
+                    + "  MarketingBudget INT64"
+                    + ") PRIMARY KEY (SingerId, AlbumId),"
+                    + "  INTERLEAVE IN PARENT Singers ON DELETE CASCADE"))
+        .get(10, TimeUnit.MINUTES);
+    databaseId = DatabaseId.of(projectId, instanceId, database);
+  }
+
+  @Before
+  public void insertTestData() {
+    final DatabaseClient client = spanner.getDatabaseClient(databaseId);
+    client.write(
+        Arrays.asList(
+            Mutation.newInsertOrUpdateBuilder("Singers")
+                .set("SingerId")
+                .to(1L)
+                .set("FirstName")
+                .to("Alice")
+                .set("LastName")
+                .to("Trentor")
+                .build(),
+            Mutation.newInsertOrUpdateBuilder("Albums")
+                .set("SingerId")
+                .to(1L)
+                .set("AlbumId")
+                .to(1L)
+                .set("AlbumTitle")
+                .to("title 1")
+                .set("MarketingBudget")
+                .to(20000L)
+                .build()));
+  }
+
+  @After
+  public void removeTestData() {
+    final DatabaseClient client = spanner.getDatabaseClient(databaseId);
+    client.write(Collections.singletonList(Mutation.delete("Singers", KeySet.all())));
+    client.write(Collections.singleton(Mutation.delete("Albums", KeySet.all())));
```

Comment:
super-nit: This is not necessary, as Albums is interleaved in Singers with `ON DELETE CASCADE`.

Reply:
Thanks for pointing this out! Removed this line of code.